src/System.Runtime.Serialization.Xml/pkg/ValidationSuppression.txt

```diff
@@ -1,2 +1,3 @@
 // Remove once we can OOB this to desktop
 PermitPortableVersionMismatch=.NETFramework,Version=v4.6.1;.NETFramework,Version=v4.6
+PermitHigherCompatibleImplementationVersion=.NETCore,Version=v5.0;.NETStandard,Version=v1.6;.NETCoreApp,Version=v1.0
```

Comment:
The implementation assembly version is not anymore matching the ref assembly. This causes a package validation error. This is, but, a known situation and we have means to suppress this error message when it's intentional. That's what this line is about; suppressing that error.